test/parallel/test-process-constants-noatime.js

```diff
@@ -1,8 +1,10 @@
+// Flags: --expose-internals
 'use strict';
 
 const common = require('../common');
 const assert = require('assert');
-const constants = process.binding('constants');
+const { internalBinding } = require('internal/test/binding');
+const constants = internalBinding('constants');
```

Comment:
I think this can just be assertions on`require('fs').constants`? Like

```
const constants = require('fs').constants;
```

And change the assertions below to something like

```
assert('O_NOATIME' in constants);
assert.strictEqual(constants.O_NOATIME, 0x40000);
```